### PR TITLE
Feature user registry

### PR DESCRIPTION
**Pull request by dievri**
Create new user and spawn their namespace on join
Load lua code from .lua

### CODE REVIEW

File: mods/core/automount.lua [hunk 1]
```diff
@@ -1,12 +1,96 @@
...
+    local cmdchan_path = tostring(core_conf:get("cmdchan_path"))
+    local root_cmdchan = cmdchan(conn, cmdchan_path)
+    if not root_cmdchan:is_present() then
+        error("cmdchan at path " .. cmdchan_path .. " is not available")
+    else
+        print("cmdchan is available")
+    end
+
+    -- mount registry
+    print(root_cmdchan:execute("mkdir -p /n/9mine /mnt/registry"))
```
Comment (metacoma) [inline mods/core/automount.lua:46]: mkdir -p /n/9mine /mnt/registry

File: mods/core/automount.lua [hunk 2]
```diff
@@ -1,12 +1,96 @@
...
+    print(root_cmdchan:execute("mount -A tcp!registry.dev.metacoma.io!30100 /mnt/registry"))
+
+    -- get and mount user management service
+    local user_management = root_cmdchan:execute("ndb/regquery -n description 'user management'"):gsub("\n", "")
+    while not user_management:match(".*!.*!.*") do
+        os.execute("sleep 2")
```
Comment (metacoma) [inline mods/core/automount.lua]: all clients will stuck for 2 seconds, use minetest.after instead of sleep

Comment (dievri) [reply mods/core/automount.lua]: Fixed. Minetest server will look for user management every 3 second with minetest.after

File: mods/core/automount.lua (hunk 1 above)
Comment (dievri) [reply mods/core/automount.lua:46]: yes, now it is with one command

**Comment by dievri** [comment]
Fixes #29

**Pull request merged**